tests/runtime/test_runtime_metrics_api.py

```diff
@@ -83,3 +84,35 @@ def test_runtime_metrics_via_env_var_manual_start(monkeypatch, ddtrace_run_pytho
 """,
     )
     assert status == 0
+
+
+def test_runtime_metrics_enable():
+    RuntimeMetrics.enable()
+    assert RuntimeWorker._instance is not None
+    assert RuntimeWorker._instance.status == ServiceStatus.RUNNING
+    assert RuntimeWorker._instance.tracer is not None
+    assert RuntimeWorker._instance.dogstatsd_url is None
+    assert RuntimeWorker._instance.interval is not None
+
+    RuntimeMetrics.disable()
+
+    tracer = DummyTracer()
+    dogstatsd_url = "udp://agent:8125"
+
+    RuntimeMetrics.disable()
+
+    RuntimeMetrics.enable(tracer=tracer, dogstatsd_url=dogstatsd_url)
+    assert RuntimeWorker._instance is not None
+    assert RuntimeWorker._instance.status == ServiceStatus.RUNNING
+    assert RuntimeWorker._instance.tracer == tracer
+    assert RuntimeWorker._instance.dogstatsd_url == dogstatsd_url
+    assert RuntimeWorker._instance.interval is not None
```

Comment:
Should we just assert the actual value here? I'd be ok with the test breaking rather than having it pass with an erroneous value like `0` or `-123`.